web/controllers/character_controller.ex

```diff
@@ -61,6 +61,8 @@ defmodule EdgeBuilder.CharacterController do
 
     render conn, :show,
       title: character.name,
+      description: description_for_character(character, user),
+      og_image_url: character.portrait_url,
```

Comment:
ARGH nevermind ignore me I am the worst. You can't avoid it because it is in the layout (which is proper)